adding lammps highlight files

terryturton opened the pull request:
@bernhold - please check the lammps<>kokkos highlight.

terryturton committed:
adding lammps highlight files

terryturton commented:
@bernhold - have you been able to review?

bernhold commented:
@terryturton it is next on my list for the CASS site.  Too many other things with actual deadlines, unfortunately

bernhold committed:
Initial edits to get story "working"

Had to do some things to get it working on the site.  Also a few edits that jumped out at me.

terryturton committed:
Updates after 2nd review of highlight

Updates after another review of lammps highlight, renaming to reflect current date

terryturton committed:
fix url error

Fixed kokkos url error

bernhold committed:
Exclude .DS_Store files from the repo

bernhold committed:
Minor edits

* Update image file names to reflect changes there
* Add teaser image back
* Move initial definition of NNSA acronym to first use

bernhold committed:
Update main page impact stories

* Add the new LAMMPS highlight
* Drop the DoD/E4S highlight
